mod/badges/set/abstract/award_badges.rb

```diff
@@ -20,10 +20,14 @@ def earns_badge action
 
 def award_badge badge_card
   name_parts = [Auth.current, badge_card.badge_type, :badges_earned]
-  badge_pointer =
-    subcard(name_parts) ||
-      attach_subcard(Card.fetch(name_parts, new: { type_id: PointerID }))
+  badge_pointer = Card.fetch(name_parts, new: { type_id: PointerID })
+  if ActManager.include? badge_pointer.name
+    director = ActManager.fetch(badge_pointer)
+    director.reset_stage
+    badge_pointer = director.card
+  end
```

Comment:
why does fetch not find the correct card?  Isn't it supposed to?

I've been thinking that the ActManager should be pretty much invisible to mod developers in most cases.  Is that how you think about it?